packages/feature-toggle/src/useFeatures/useFeature.tsx

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react'
+import FeaturesContext from '../FeaturesContext'
+
+const useFeatures = <T extends object>() => React.useContext<T>(FeaturesContext)
```

Comment:
should we use param here to get the feature we want rather than returning the entire array

Reply:
There are 2 variants that works right now
```
// 1 ------
const useFeature <T extends object, K extends keyof T>(featureName: K) 
    => React.useContext<T>(FeaturesContext)[featureName]
...
useFeature<AppFeatures, 'MyAwesomeFeature'>('MyAwesomeFeature')


// 2 ------
const useFeatures <T extends object>() 
    => React.useContext<T>(FeaturesContext)
...
useFeatures<AppFeatures>()['MyAwesomeFeature']
```

So I decided to go with the 2nd choice